code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -455,10 +455,37 @@
 //Abductors
 /datum/admins/proc/makeAbductorTeam()
 	new /datum/round_event/ghost_role/abductor
-	return 1
+	return TRUE
 
 /datum/admins/proc/makeRevenant()
 	new /datum/round_event/ghost_role/revenant(TRUE, TRUE)
-	return 1
+	return TRUE
+
+/datum/admins/proc/makeNerd()
+	var/spawnpoint = pick(GLOB.blobstart)
+	var/list/mob/dead/observer/candidates
+	var/mob/dead/observer/chosen_candidate
+	var/mob/living/simple_animal/drone/nerd
+	var/teamsize
+
+	teamsize = input(usr, "How many drones ?", "N.E.R.D. team size", 2) as num|null
+
+	if(teamsize <= 0)
+		return FALSE
+
+	candidates = pollGhostCandidates("Do you wish to be considered for a nanotrasen emergency response drone?", "Drone")
+
+	if(length(candidates) == 0)
+		return FALSE
+
+	while(length(candidates) && teamsize)
+		chosen_candidate = pick(candidates)
+		candidates -= chosen_candidate
+		nerd = new /mob/living/simple_animal/drone(spawnpoint)
```

Comment:
should probably give these drones custom laws

Reply:
They kind of crawl out of the vents so I think standard laws fit fine.